Combat/Include/Points.hpp

```diff
@@ -4,14 +4,17 @@
 struct Points
 {
    uint _maxPoints;
+private:
```

Comment:
Czemu _maxPoints jest publiczne?

Zawsze publiczny interfejs dajemu do góry, a prywatny na dole. Jeśli implementujemy tutaj większość funkcji typu add, remove itd to zmieniamy na klasę. struct używamy raczej do struktur.